Reneiski_Homework1/ReneiskiSelenium11/Tests/BaseTest.cs

```diff
@@ -0,0 +1,30 @@
+﻿using NUnit.Framework;
+using OpenQA.Selenium;
+using OpenQA.Selenium.Chrome;
+using OpenQA.Selenium.Interactions;
+using OpenQA.Selenium.Support.UI;
+
+namespace ReneiskiSelenium11.Tests
+{
+    public class BaseTest
+    {
+        protected IWebDriver _driver;
+        protected WebDriverWait _driverWait;
+        protected Actions _driverActions;
+
+        [OneTimeSetUp]
+        public void OneTimeSetup()
+        {
+            _driver = new ChromeDriver();
+            _driver.Manage().Window.Maximize();
+            _driverWait = new WebDriverWait(_driver, TimeSpan.FromSeconds(1000));
```

Comment:
I would say some improvements are needed there:
1. 1000 seconds is too big timeout
2. It's better to implement webdriver factory so as to manage browsers
3. implement webdriver extension with method GetWebDriverWait